test_fms/mpp/test_mpp_domains2.sh

```diff
@@ -39,7 +39,7 @@ echo "1: Test simple functionality"
 run_test test_domains_simple 4 
```

Comment:
This is still failing in gaea. `touch input.nml` needed to be added before you run the test.

Reply:
I agree. I will get it moved.